### DISCUSSION TITLE
CODM keymapping

### DISCUSSION THREAD

**Discussion by SabunMndi** [barry-ran/QtScrcpy#866] [Q&A] [answered]
do you have keymapp for codm or can you tell me how to make it?

**Comment by SabunMndi** [comment]
never mind i just made it my self

**Comment by SabunMndi** [comment]
[Codm Multiplayer 2.json](https://github.com/barry-ran/QtScrcpy/files/13193488/Codm.Multiplayer.2.json)

i just made it my self its not perfect because i just learn how to do it

**Reply by SabunMndi** [reply, marked as answer]
but you have to use my controller layout in codm
kode to the layout 7282577673726944045-7283936317910862418

**Reply by carlopogi088** [reply]
bro your script is not complete doesn't have a switch weapon do you have an updated one or can you teach me how

**Reply by SabunMndi** [reply]
you can try using notepad++ for make the script just follow what i make in my script

**Reply by niche710** [reply]
@SabunMndi hello, how to add the left shift Key as walk in CODM? I can't find anything on web. Thank you

**Reply by niocresup** [reply]
how do you create key map?

**Comment by w4po** [comment]
Hi, Sorry for the late response but I've just made a visual key mapper for QtScrcpy: https://github.com/w4po/ScrcpyKeyMapper
